ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -128,6 +140,24 @@ open class AvatarView: UIView {
         }
     }
 
+    private var frameView: UIView? {
```

Comment:
customBorderView? customBackgroundView?

Reply:
could we just user borderView and combine the logic?

Reply:
agree name is not great. I just wanted it to be clear that this is a thicker border, almost like a picture frame. But the work frame in code means something different... I think I'll go with customBorderView.